### PR TITLE
Split the long Content.ModeKind texts into lines

### PR DESCRIPTION
**Pull request by kayvank**
This pull request is a fix for issue-283.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## [v0.11.0.0](https://github.com/LambdaHack/LambdaHack/compare/v0.10.3.0...v0.11.0.0)
 
+- Split the long Content.ModeKind texts into lines, [issue 283](https://github.com/LambdaHack/LambdaHack/issues/283)
```
Comment (Mikolaj) [inline CHANGELOG.md:3]: Thank you for thinking about the changelog. LambdaHack v0.11.0.0 has already been released. Please start a new section (v0.11.1.0) and move your item there.

File: GameDefinition/Content/ModeKind.hs [hunk 2]
```diff
@@ -125,7 +125,11 @@ brawl = ModeKind  -- sparse melee in daylight, with shade for melee ambush
...
-  , mhint   = "Run a short distance with Shift or LMB, switch the pointman with Tab, repeat. In open terrain, if you keep distance between teammates, this resembles the leap frog infantry tactics. For best effects, end each sprint behind a cover or concealment.\nObserve and mimic the enemies. If you can't see an enemy that apparently can see you, in reversed circumstances you would have the same advantage. Savour the relative fairness --- you won't find any in the main crawl adventure that follows.\nIf you get beaten repeatedly, try using all consumables you find. Ponder the hints from the defeat message, in particular the one about keeping your party together once the opponents are spotted. However, if you want to discover a winning tactics on your own, make sure to ignore any such tips until you succeed."
+  , mhint   =  T.intercalate "\n"
+    ["Run a short distance with Shift or LMB, switch the pointman with Tab, repeat. In open terrain, if you keep distance between teammates, this resembles the leap frog infantry tactics. For best effects, end each sprint behind a cover or concealment."
```
Comment (Mikolaj) [inline GameDefinition/Content/ModeKind.hs:129]: A space between `[` and `"Run` would give it a more uniform style, given the space between comma and text below. And similarly elsewhere.

File: GameDefinition/Content/ModeKind.hs [hunk 3]
```diff
@@ -264,7 +283,11 @@ ambush = ModeKind  -- dense ranged with reaction fire vs melee at night
   , mdesc   = "Prevent hijacking of your ideas at all cost! Be stealthy, be observant, be aggressive. Fast execution is what makes or breaks a creative team."
   , mreason = "In this adventure, finally, your heroes are able to all use ranged attacks at once, given enough ammunition."
   , mhint   = "Beware of friendly fire, particularly from explosives. But you need no more hints. Go fulfill your destiny! For Science!"
-  }
+  } where
```
Comment (Mikolaj) [inline GameDefinition/Content/ModeKind.hs:286]: I think the formatting with `where` on a separate one that you used before, is better.

**Review by Mikolaj — APPROVED** [review]
Well done.

**Comment by kayvank** [comment]
Thank you @Mikolaj for reviewing,  approving, and helping me with this PR. 
I've implemented your suggestions. Is there anything else I need to do to get the "workflow approval" required for the merge?

**Comment by Mikolaj** [comment]
No, the "workflow approval" is because you are first-time commiter. Your next PR should run CI automatically, without any approval needed. As soon as CI finishes, I will squash and merge. BTW, you could have amended the existing commit (`git commit --amend -a`) and done `push -f`. But no problem, squash and merge will make these commits into one.

**Pull request merged**